source/util/AMSettings.cpp

```diff
@@ -171,9 +170,9 @@ void AMSettings::load() {
 	publicDataFolder_ = settings.value("publicDataFolder", "/home/acquaman/data/").toString();
 	publicDatabaseFilename_ = settings.value("publicDatabaseFilename", "publicdata.db").toString();
 
-	QString defaultBasePath(QDir::homePath() % "/beamline/programming");
-	fileLoaderPluginsFolder_ = settings.value("fileLoaderPluginsFolder", QString(defaultBasePath % "/acquaman/plugins/FileLoaders")).toString();
-	analysisBlockPluginsFolder_ = settings.value("analysisBlockPluginsFolder", QString(defaultBasePath % "/acquaman/plugins/AnalysisBlocks")).toString();
+	QString defaultBasePath(QCoreApplication::applicationDirPath() % "/..");
```

Comment:
@dretrex Both solutions are based on the assumption that the plugin folder is in the same super folder of the executable ones. The only difference is that with QFileInfo, we can check whether it is a valid path or not; when we need to handle the case when the path is invalid.

I can use QFileInfo for sure.